packages/common/src/store/buy-crypto/slice.ts

```diff
@@ -0,0 +1,82 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { MintName } from 'services/index'
+import { OnRampProvider } from 'store/ui/buy-audio/types'
+
+type BuyCryptoPayload = {
+  /**
+   * The amount the user is requesting, in user friendly decimal denomination
+   */
+  amount: number
+  /**
+   * The mint name of the token the user wants to purchase
+   */
+  mint: MintName
+  /**
+   * The service used to purchase the SOL necessary
+   */
+  provider: OnRampProvider
+}
+
+type BuyCryptoState = {}
```

Comment:
No state tracking at all?

Reply:
left the skeletons here if we wanted to add in the future. I didn't see the need right now.